homeassistant/components/binary_sensor/zwave.py

```diff
@@ -94,7 +94,8 @@ def should_poll(self):
 
     def value_changed(self, value):
         """Called when a value has changed on the network."""
-        if self._value.value_id == value.value_id:
+        if self._value.value_id == value.value_id or \
```

Comment:
You should be able to do `self._value.node == value.node` right ? Because that test is also true when value ids are the same?